docs/operator-manual/applicationset.yaml

```diff
@@ -1,38 +1,379 @@
 apiVersion: argoproj.io/v1alpha1
 kind: ApplicationSet
+data:
+  # Within Kubernetes these fields are actually encoded in Base64; they are decoded here for convenience.
+  # (They are likewise decoded when passed as parameters by the Cluster generator)
+  config: "{'tlsClientConfig':{'insecure':false}}"
+  name: "in-cluster2"
+  server: "https://kubernetes.default.svc"
+
 metadata:
   name: test-hello-world-appset
   namespace: argocd
+ 
+  # To preserve this annotation and label we can use the preservedFields property 
+  preservedFields:
+    # This annotation and label exists only on this Application, and not in 
+    # the parent ApplicationSet template:
+    annotations: 
+      my-custom-annotation: some-value
+    labels:
+      my-custom-label: some-value
```

Comment:
I'd add a note that `ignoreApplicationDifferences` is the preferred way to accomplish this now.